### ISSUE TITLE
Think about migration

### ISSUE THREAD

**Issue by Procrat** [ZeusWPI/zeus.ugent.be#78]
How should old URLs be handled?

**Comment by feliciaan** [comment]
Three options:
1. Do nothing, except for the 404 page which says their is a new site
2. Redirect to the old site
3. List all old links, and create redirects. (maybe scripted?)

**Comment by wschella** [comment]
I'd keep the old site up and running for redirects.

**Comment by werthen** [comment]
Redirecting old blogposts would be nice

**Comment by feliciaan** [comment]
@werthen can you look in the logs for zeus.ugent.be, and count the 404s?
Maybe group and count them.

**Comment by werthen** [comment]
Closing this issue, we should have thought some things out beforehand, but the site has been running for over a year and I can't really find any 404's not by crawlers/spiders.

**Comment by wschella** [comment]
I think it might still be useful to go trough some old logs, and check for some patterns.
i'm assuming multiple blog-posts will not resolve, which might be sad, although not a disaster.
For the sake of continuity, let's not close this, just keep it as low priority.

**Comment by Procrat** [comment]
You're right. It was frustrating at the time, but by now it's just history, which is probably largely irrelevant.

@werthen: I don't know what crawler you used, but I'm finding more than a hundred 404's, mainly external ones. Here's a handful that are internal:
 - 404 https://zeus.ugent.be/blog/09-10/agenda-oktober/ → https://zeus.ugent.be/2009/09/23/binnenkort-in-gent/
 - 404 https://zeus.ugent.be/blog/09-10/devteam-meeting-wina/ → https://zeus.ugent.be/2009/09/23/summer-of-code/
 - 404 https://zeus.ugent.be/blog/09-10/django-on-rails/ → https://zeus.ugent.be/2009/10/02/agenda-oktober/
 - 404 https://zeus.ugent.be/blog/09-10/django-on-rails/ → https://zeus.ugent.be/over-zeus-wpi/waar-vind-je-ons/
 - 404 https://zeus.ugent.be/blog/09-10/karmic-koala/ → https://zeus.ugent.be/kelder/
 - 404 https://zeus.ugent.be/blog/10-11/review-tapworthy/ → https://zeus.ugent.be/kelder/bibliotheek/
 - 404 https://zeus.ugent.be/blog/12-13/bestuur-2013-2014/ → https://zeus.ugent.be/contact/
 - 404 https://zeus.ugent.be/blog/12-13/ubuntu-op-dell-vostro-3560-en-dell-precision-m4700/ → https://zeus.ugent.be/2013/07/28/ubuntu-op-dell-vostro-3560-en-dell-precision-m4700/
 - 404 https://zeus.ugent.be/blog/13-14/zeus-bestuur-2014-2015/ → https://zeus.ugent.be/contact/
 - 404 https://zeus.ugent.be/blog/15-16/badass-battle-bots-is-back/ → https://zeus.ugent.be/2016/02/26/badass-battle-bots/
 - 404 https://zeus.ugent.be/events/07-08/ → https://zeus.ugent.be/logo.png
 - 404 https://zeus.ugent.be/events/10-11/fosdem-2011/ → https://zeus.ugent.be/2009/12/07/fosdem-2010/
 - 404 https://zeus.ugent.be/events/11-12/vlaamse-programmeerwedstrijd-2012/ → https://zeus.ugent.be/2012/02/20/vlaamse-programmeerwedstrijd-2012/
 - 404 https://zeus.ugent.be/events/11-12/vlaamse-programmeerwedstrijd-2012/ → https://zeus.UGent.be/contact/
 - 404 https://zeus.ugent.be/events/11-12/windows-phone-7-5-build-an-application-in-4-hours/ → https://zeus.ugent.be/2009/10/27/introductie-tot-cocoa-development/
 - 404 https://zeus.ugent.be/events/11-12/windows-phone-7-5-build-an-application-in-4-hours/ → https://zeus.ugent.be/2010/10/22/android-development-introductie/
 - 404 https://zeus.ugent.be/events/11-12/windows-phone-7-5-build-an-application-in-4-hours/ → https://zeus.ugent.be/2011/10/21/windows-phone-7-5-build-an-application-in-4-hours/
 - 404 https://zeus.ugent.be/events/13-14/1650/ → https://zeus.ugent.be/2013/07/12/codenights/
 - 404 https://zeus.ugent.be/events/13-14/1650/ → https://zeus.ugent.be/over-zeus-wpi/waar-vind-je-ons/
 - 404 https://zeus.ugent.be/events/13-14/geeks-attack/ → https://www.zeus.ugent.be/lan/lan-praktisch
 - 404 https://zeus.ugent.be/events/13-14/google-summer-of-code-2014/ → https://zeus.ugent.be/contact/
 - 404 https://zeus.ugent.be/events/14-15/geeks-attack-2014/ → https://zeus.ugent.be/truien/


A bunch of other things came to light as well.

## `kelder.zeus.ugent.be`
These links are down since the user dirs on `kelder.zeus.ugent.be` have gone down
 - 404 https://zeus.ugent.be/blog/08-09/the-importance-of-subnets/ → http://kelder.zeus.ugent.be/~blackskad/blog/
 - 404 https://zeus.ugent.be/blog/08-09/the-importance-of-subnets/ → http://kelder.zeus.ugent.be/~blackskad/blog/wp-includes/images/smilies/icon_smile.gif
 - 404 https://zeus.ugent.be/events/07-08/introductie-tot-latex/ → http://kelder.zeus.ugent.be/~gaspard/latex/
 - 404 https://zeus.ugent.be/events/12-13/collaborate-with-git/ → https://kelder.zeus.ugent.be/~jasper/git-talk.html
 - 404 https://zeus.ugent.be/events/14-15/python-voor-gevorderden/ → https://kelder.zeus.ugent.be/~stijns/python-indoctrinatie.pdf

## `www.zeus.ugent.be`
The host name in the certificate is invalid or does not match.
 - https://zeus.ugent.be/events/12-13/gamers-assemble/ → https://www.zeus.ugent.be/lan/
  - https://zeus.ugent.be/events/13-14/geeks-attack/ → https://www.zeus.ugent.be/lan/lan-praktisch

## Bottle Bats
Links to the old Badass Bottle Bats don't redirect to the new repo.
 - 500 https://zeus.ugent.be/blog/15-16/badass-battle-bots/ → https://zeus.ugent.be/bottlebats
 - 500 https://zeus.ugent.be/blog/15-16/badass-battle-bots-is-back/ → https://zeus.ugent.be/bottlebats/
 - 500 https://zeus.ugent.be/projects/ → https://zeus.ugent.be/bottlebats/

## Haldis
These look like a bug in Haldis.
 - 404 https://zeus.ugent.be/haldis/location/17 → https://zeus.ugent.be/haldis/location/None
 - 404 https://zeus.ugent.be/haldis/location/19 → https://zeus.ugent.be/haldis/location/TZIJN ENKEL KAPSALONS
 - 404 https://zeus.ugent.be/haldis/location/2 → https://zeus.ugent.be/haldis/location/None
 - 404 https://zeus.ugent.be/haldis/location → https://zeus.ugent.be/haldis/None
 - 404 https://zeus.ugent.be/haldis/location → https://zeus.ugent.be/haldis/TZIJN ENKEL KAPSALONS

## Gamification
These look like a bug in Gamification.
 - 404 https://zeus.ugent.be/game → https://zeus.ugent.be/game/repositories/zeus.ugent.be
 - 404 https://zeus.ugent.be/game/repositories → https://zeus.ugent.be/game/repositories/z.e.u.s.

## 12urenloop
Host name in the certificate is invalid or doesn't match.
 - https://zeus.ugent.be/blog/10-11/counting-laps-using-bluetooth-dongle-detection-on-the-12-urenloop/ → https://www.12urenloop.be/
